arbnode/transaction_streamer.go

```diff
@@ -1612,664 +1533,3 @@ func (s *TransactionStreamer) Start(ctxIn context.Context) error {
 
 	return stopwaiter.CallIterativelyWith[struct{}](&s.StopWaiterSafe, s.executeMessages, s.newMessageNotifier)
 }
-
-// Encodes an uint64 as bytes in a lexically sortable manner for database iteration.
-
-func (s *TransactionStreamer) BlockMetadataAtCount(count arbutil.MessageIndex) (common.BlockMetadata, error) {
```

Comment:
nit: why are we deleting this function `BlockMetadataAtCount`? These seems like a function from offchains labs and should probably remain in the transaction_streamer file

Reply:
This function seems to no longer be referenced.

The way we are handling the upstream changes is attributing the authors of these functions to our team rather than the actual authors of the commits and leads to confusion about who is responsible for what line of code.

This method was utilized in `v3.5.2`, and seems to no longer be referenced in `v3.6.7` with the introduction of the method `BlockMetadataAtMessageIndex`.

If it's not out code, I'm fine with leaving it in.  But we really need to figure out a better way to keep the line attribution as is, instead of re-authoring all of these commits.